packages/style-engine/class-wp-style-engine.php

```diff
@@ -136,6 +136,20 @@ final class WP_Style_Engine {
 				),
 			),
 		),
+		'dimensions' => array(
```

Comment:
This is a good top-level label. 

Carry on.

Reply:
Side note: I'm working on a PR that will allow us to extend base style definitions

- https://github.com/WordPress/gutenberg/pull/45296

Nothing to do here. I think having a set of definitions for `dimensions: { width, height, ... }` in the base set of definitions makes sense 👍

Reply:
When the time comes, after testing this branch and so on, what do you think about splitting the style engine additions into a dependency PR? 

I'm just thinking of the need for unit tests and how to avoid bloating this PR.

Reply:
> what do you think about splitting the style engine additions into a dependency PR?

Thanks for taking a look!

Absolutely, I'm all for splitting this up into component parts to make the merge and review process easier. It's super easy for these new block support addition PRs to become unwieldy 👍

Reply:
Splitting this up a bit sounds like a solid plan. Might make tracking down when individual blocks had their bespoke controls moved over to this block support easier as well.